src/components/DesktopPatientTable/index.js

```diff
@@ -150,6 +197,27 @@ const DesktopPatientTable = (props) => {
                   <Label>Last updated:</Label>
                   <Value>{getFormatedTimeDate(patient?.lastUpdated)}</Value>
                 </Info>
+                <ButtonWrap className="ml-2 ">
+                  <Button
+                    className="btn btn-download-small"
+                    onClick={() => {
+                      exportVitals(patient.patientId);
+                    }}>
+                    <span className="excel-image-wrap-small">
+                      <img
+                        src={excel}
+                        alt="Covin"
+                        className="logo download-excel-icon-small"
+                      />
+                      <img
+                        src={xicon}
+                        alt="Covin"
+                        className="logo x-icon-small"
+                      />
+                    </span>{' '}
```

Comment:
please replace {' '} with styling for span

Reply:
{' '} removed